LAYAIraq/LAYA-Iraq-Frontend
# Register hint

Updated register.vue to make hints clearer.

## What to test:
- check register view with wrong inputs for name and mail
- see if hints are more obvious
